src/app/components/user-details/user-details.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="user-details">
+    <form [formGroup]="userFormGroup">
+        <div>
+            <label for="username">Username:</label><br>
+            <input type="text" formControlName="username">
```

Comment:
keep form control namign consistent, so if you use first_name aka snake_case then use it for all variables throughout the project if you use camelCase then use firstName etc. keep it consistent